pyqtgraph/graphicsItems/PlotCurveItem.py

```diff
@@ -801,10 +918,25 @@ def paint(self, p, opt, widget):
         if self.xData is None or len(self.xData) == 0:
             return
 
-        if getConfigOption('enableExperimental'):
-            if HAVE_OPENGL and isinstance(widget, QtWidgets.QOpenGLWidget):
-                self.paintGL(p, opt, widget)
-                return
+        do_fill = self.opts['brush'] is not None and self.opts['fillLevel'] is not None
+        do_fill_outline = do_fill and self.opts['fillOutline']
+
+        if (
+            getConfigOption('enableExperimental')
```

Comment:
do you think we should hide this functionality being the `enableExperimental` config option at this point? The downside of doing this is that it won't get as much usage and we may not get any feedback on its functionality.

Reply:
#2257 shows that there's 1 user using OpenGL while simultaneously leaving `enableExperimental=False`. So at this point, enabling `paintGL` codepath w/o `enableExperimental=True` may constitute an API breakage for them.
@bbc131, would you care to chime in on this?

Reply:
> #2257 shows that there's 1 user using OpenGL while simultaneously leaving `enableExperimental=False`. So at this point, enabling `paintGL` codepath w/o `enableExperimental=True` may constitute an API breakage for them. @bbc131, would you care to chime in on this?

We don't use this configuration (OpenGL and `enableExperimental=False`) anymore, so this won't break anything for us.